docker-distribution/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM gradle:3.5-jre8-alpine
+MAINTAINER varkockova.a@gmail.com
+
+USER root
+COPY build.gradle /
+WORKDIR /
+ARG PLUGIN_VERSION
+# ARGs are not persisted so we must store value of ARG to ENV
+ENV PLUGIN_VERSION=$PLUGIN_VERSION
```

Comment:
I would prefer if this was a ARG and new tagged version of docker image will be released for every new release of the plugin. What do you think?